src/components/application_manager/rpc_plugins/sdl_rpc_plugin/src/commands/hmi/sdl_activate_app_request.cc

```diff
@@ -139,7 +145,7 @@ void SDLActivateAppRequest::Run() {
   const uint32_t application_id = app_id();
 
   ApplicationConstSharedPtr app_to_activate =
-      application_manager_.application(application_id);
+      application_manager_.application_by_hmi_app(application_id);
```

Comment:
For regular apps, the application_id is the internal app id, not the hmi app id. Core can't find the application when activating a normal app.